requirements/pip.in

```diff
@@ -0,0 +1 @@
+pip-tools==5.5.0
```

Comment:
Why it should be in a separate file?
I would leave it in `dev.in`

Reply:
I moved pip-tools to a separate file because the CI doesn't need pip-tools and installing them means extra work that's not needed for the CI.

Reply:
Left it in `dev.in` again via aa42469.